frontend/src/scenes/trends/persons-modal/persons-modal-utils.tsx

```diff
@@ -53,9 +55,29 @@ export const pathsTitle = (props: { mode: pathModes; label: string }): React.Rea
 }
 
 export const cleanedInsightActorsQueryOptions = (
-    insightActorsQueryOptions: InsightActorsQueryOptionsResponse | null
+    insightActorsQueryOptions: InsightActorsQueryOptionsResponse | null,
+    query: InsightActorsQuery
 ): [string, any[]][] => {
-    return Object.entries(insightActorsQueryOptions ?? {}).filter(([, value]) => {
+    const cleanedOptions = Object.entries(insightActorsQueryOptions ?? {}).filter(([, value]) => {
         return Array.isArray(value) && !!value.length
     })
+    const source = query?.source
+    const seriesNames = isTrendsQuery(source) ? source.series.map((s: any) => s.custom_name) : []
+    const cleanedOptionsWithAdjustedSeriesNames: [string, any[]][] = cleanedOptions.map(([key, value]) => {
+        if (key === 'series') {
+            return [
+                key,
+                value.map((v: any, index: number) => ({
+                    ...v,
+                    label:
+                        seriesNames[index] ??
+                        getCoreFilterDefinition(v.label, TaxonomicFilterGroupType.Events)?.label ??
```

Comment:
An extra bit of missing formatting I thought is worth adding: We'll now also format PostHog core events (prefixed with $), e.g. "$pageview" becomes "Pageview"